white_label/brands/mabulle/js/locales/extra-es.json

```diff
@@ -1,5 +1,23 @@
 {
+  "logout_dialog": {
+    "content": "¿Estás seguro de que quieres desconectar esta aplicación de tu espacio personal?"
```

Comment:
Regarding the wording of cozy, the use of "espacio personal" make sens, just a little nit :  usually we would use "su/tu cuenta", so it could be discuss, as you wish 👍